pkg/infradb/taskmanager/taskmanager.go

```diff
@@ -103,7 +105,15 @@ func (t *TaskManager) StatusUpdated(name, objectType, resourceVersion, notificat
 	log.Printf("StatusUpdated(): New Task Status has been created and sent to channel: %+v\n", taskStatus)
 }
 
+// ReplayFinished notifies that the replay of objects has finished
+func (t *TaskManager) ReplayFinished() {
+	close(t.replayChan)
+	log.Println("ReplayFinished(): Replay has finished.")
+}
+
 // processTasks processes the task
+//
+//gocognit:ignore
```

Comment:
I suppose it won't take much time to create a dedicated function rather than suppress warnings

Reply:
Let me see what I can do for this

Reply:
Done